OpenKh.Tools.Kh2TextEditor/ViewModels/TextEditorViewModel.cs

```diff
@@ -127,6 +139,66 @@ public TextEditorViewModel()
             Drawing = new SpriteDrawingDirect3D();
             CurrentMessageEncoder = Encoders.InternationalSystem;
             _messages = new MessagesModel(this, this, new Msg.Entry[] { });
+
+            HandleReturn = new RelayCommand(x =>
+            {
+                Text += "{:newline}";                
+                SelectedItem = SelectedItem;
```

Comment:
This is a bit confusing, even if it works. I have few suggestions to improve it:
On `public string Text` setter, at the very end we can use `OnPropertyChanged()`, so there is no need for a `SelectedItem = SelectedItem`. I also realise that having the `{:newline}` appended at the end regardless the position of the cursor is not ideal...

What do you think if we just get rid of `{:newline}` to replace it with a `\n`? There is an unit test called `DeserializeNewLine` that is responsible to double-check how the new-line is handled. As long as you change it to `Assert.Equal("hello\nworld", commands[0].Text);` and the test is green, the functionality will be implemented.

Reply:
Well, OnPropertyChanged() changed the string, but not the TextBox for some reason. I had to compromise.
And yes, /n instead of {:newline} would be better imho.